### PR TITLE
Add shared script to reload images

### PR DESCRIPTION
**Pull request by mkolesnik**
The script allows reloading images and restarting the necessary deployments/daemonsets.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,20 +11,18 @@ export BASE_BRANCH OCM_BASE_BRANCH
...
-CLUSTER_SETTINGS_FLAG = --settings $(DAPPER_SOURCE)/.shipyard.e2e.yml
+SETTINGS ?= $(DAPPER_SOURCE)/.shipyard.e2e.yml
 endif
 
-override CLUSTERS_ARGS += $(CLUSTER_SETTINGS_FLAG)
-override DEPLOY_ARGS += $(CLUSTER_SETTINGS_FLAG)
 override E2E_ARGS += --nolazy_deploy cluster1
 
+# Running in Dapper
+
```
Comment (skitt) [inline Makefile]: This should introduce the whole “running in Dapper” section, *i.e.* lines 14 and following.

**Review by skitt — CHANGES_REQUESTED** [review]
Minor comment re a comment ;-).

File: Makefile (hunk 1 above)
Comment (mkolesnik) [reply Makefile]: Done

**Review by skitt — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -13,18 +13,16 @@ ifneq (,$(DAPPER_HOST_ARCH))
...
+SETTINGS ?= $(DAPPER_SOURCE)/.shipyard.e2e.yml
 endif
 
-override CLUSTERS_ARGS += $(CLUSTER_SETTINGS_FLAG)
-override DEPLOY_ARGS += $(CLUSTER_SETTINGS_FLAG)
 override E2E_ARGS += --nolazy_deploy cluster1
 
+include Makefile.inc
+
 # Prevent rebuilding images inside dapper since thy're already built outside it in Shipyard's case
```
Comment (nyechiel) [inline Makefile]:
```suggestion
# Prevent rebuilding images inside dapper since they're already built outside it in Shipyard's case
```

**Review by nyechiel — COMMENTED** [review]
@mkolesnik one nit (typo). Can you also ensure the we document this under https://submariner.io/development/building-testing/?

**Review by nyechiel — APPROVED** [review]

**Review by sridhargaddam — APPROVED** [review]

**Comment by nyechiel** [comment]
@mkolesnik any idea what's up with the failing tests here?

**Comment by nyechiel** [comment]
@mkolesnik a reminder about docs

**Pull request merged**